src/index-fn.js

```diff
@@ -0,0 +1,14 @@
+export {default as getColor} from "./getColor.js";
```

Comment:
If we replace `export default` to `export` in `./*.js` this file will be smaller.

Reply:
Yes but default exports are nicer for the individual modules.